README.md

```diff
@@ -3,17 +3,42 @@
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/runtime-builder-java.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/runtime-builder-java)
 
 A [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/) pipeline for 
-packaging Java applications into supported Google Cloud Runtime containers. It consists of several
-docker containers which are run using the [cloudbuild.yaml](cloudbuild.yaml) configuration file.
+packaging Java applications into supported Google Cloud Runtime containers. It consists of a series
+of docker containers and a [cloudbuild.yaml](cloudbuild.yaml) configuration file.
 
-## Building
-To build all docker images:
+## Running via Google Cloud Container Builder (recommended)
+To run via Google Cloud Container Builder, first install the 
+[Google Cloud SDK](https://cloud.google.com/sdk/). Then, initiate a Cloud Container Build:
 ```bash
-mvn package
+gcloud container builds submit /path/to/my/java/app --config cloudbuild.yaml
```

Comment:
Say something like, "After this command is run, the built application container will appear in GCR..."